extensions-contrib/statsd-emitter/src/main/resources/defaultMetricDimensions.json

```diff
@@ -175,5 +175,5 @@
   "namespace/cache/numEntries" : { "dimensions" : [], "type" : "gauge" },
   "namespace/cache/heapSizeInBytes" : { "dimensions" : [], "type" : "gauge" },
 
-  "service/heartbeat" : { "dimensions" : ["leader"], "type" : "gauge" }
+  "service/heartbeat" : { "dimensions" : ["leader"], "type" : "count" }
```

Comment:
This should be gauge IMO, as it's not a cumulative value. E.g. it doesn't make sense to say the total number of `heartbeat` yesterday.

Reply:
From the [datadog docs](https://docs.datadoghq.com/metrics/types/?tab=count#metric-types) - it looks like a count metric reports the count of the metrics retrieved in that interval and is not cumulative across intervals.

```
Suppose you are submitting a COUNT metric, activeusers.basket_size, from a single host running the Datadog Agent. This host emits the following values in a flush time interval: [1,1,1,2,2,2,3,3].

The Agent adds all of the values received in one time interval. Then, it submits the total number, in this case 15, as the COUNT metric’s value.
```

From an analysis point, it is very useful to know how many heartbeats were sent yesterday as it gives us an idea of uptime if we know how many heartbeats are sent per minute.